apps/trail/gpx2egt.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+grep "trkpt lat" | sed 's/.*trkpt.lat=.//' | sed 's/. lon=./ /' | sed 's/".$//'
```

Comment:
In case the `<trkpt>` has extra attributes, this would pick out just lat/lon - will leave it up to you though :)

```suggestion
grep "<trkpt" | grep -Eo '(lat|lon)="[^"]*"' | tr -d '"' | sed 's/.*=//' | paste -d' ' - -
```

Reply:
Yes, better gpx parsing would be cool, I'm sure there's good library somewhere. Ideally GPX would be uploaded from web interface, but command line is just easier for me.

Feel free to merge either version, so I can merge it and continue development.

Reply:
Not a problem - we can leave this for a potential html version in the future